### PR TITLE
VSFP-21: Apply restyling according to Haskell Substitution Stepper

### PR DESCRIPTION
**Pull request by fl3pp**

### CODE REVIEW

File: SA/sa.typ [hunk 1]
```diff
@@ -3,13 +3,25 @@
...
-  "Designing a Visual, Block-Based Environment to Create & Execute Haskell Code");
+#let metadata = (
+  title: [VisualFP],
+  description: [Designing a Visual, Block-Based Environment to Create & Execute Haskell Code],
+  organization: [Department of Computer Science \ OST - University of Applied Sciences \ Campus Rapperswil-Jona],
+  thesis: [Student Thesis],
+  term: [Autumn Term 2023],
+  authors: [Lukas Streckeisen, Jann M. Flepp],
+  authors-short: [L. Streckeisen, J. Flepp],
+  advisor: [Farhad D. Mehta],
```
Comment (lstreckeisen) [inline SA/sa.typ:14]:
```suggestion
  advisor: [Prof. Dr. Farhad D. Mehta],

```
I think in an academic environment this is kind of important ;)

File: SA/sa.typ [hunk 2]
```diff
@@ -3,13 +3,25 @@
...
 #set heading(..sa_header_style)
 
-#sa_title_page(
-  "VisualFP",
-  "Designing a Visual, Block-Based Environment to Create & Execute Haskell Code");
+#let metadata = (
+  title: [VisualFP],
+  description: [Designing a Visual, Block-Based Environment to Create & Execute Haskell Code],
+  organization: [Department of Computer Science \ OST - University of Applied Sciences \ Campus Rapperswil-Jona],
+  thesis: [Student Thesis],
```
Comment (lstreckeisen) [inline SA/sa.typ:10]:
```suggestion
  thesis: [Term Paper],

```
The DeepL translation of "Studienarbeit" would be student research project, but I like the less common translation "term paper" better

File: SA/sa.typ (hunk 1 above)
Comment (fl3pp) [reply SA/sa.typ]: Yes, it is 😊

Adjusted it accordingly

File: SA/sa.typ (hunk 2 above)
Comment (fl3pp) [reply SA/sa.typ]: Sounds good 👍

**Pull request merged**